.style-and-markdown-build.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+ret=0
+
+if grep -rn $'\t' modules assemblies pom.xml --include=pom.xml; then
+  echo "Tabs found!"
+  ret=1
+fi
+if grep -rn ' $' modules assemblies pom.xml --include=pom.xml; then
+  echo "Trailing spaces found!"
+  ret=1
+fi
+if grep -rn '	' etc; then
+  echo "Tabs found in config files!"
+  ret=1
+fi
+if grep -rn ' $' etc; then
+  echo "Trailing spaces found in config files!"
+  ret=1
+fi
+
+
+# build number must be present in all modules
+if ! grep -L '<Build-Number>${buildNumber}</Build-Number>' modules/*/pom.xml | wc -l | grep -q '^0$'; then
+  echo "Build number is missing from a module!"
+  ret=1
+fi
+
+# maven-dependency-plugin should be active for all new modules
+#grep -L maven-dependency-plugin modules/*/pom.xml > maven-dependency-plugin.list
+#if ! diff -q maven-dependency-plugin.list docs/checkstyle/maven-dependency-plugin.exceptions; then
+#  rev=1
+#fi
+
+cd docs/guides
+
+for docs in admin developer user; do
+  pushd . > /dev/null
+  cd $docs
```

Comment:
`pushd` stores the original directory, making the additional `cd` somewhat weird.
Using this would work the same:
```sh
pushd "$docs"
```

But since you are not keeping multiple locations, you can also just use the simpler `cd; …; cd -` which also does not print making the pipe unnecessary.

```suggestion
  cd "$docs"
```